README.md

```diff
@@ -18,19 +18,29 @@ This Rails app is most easily run and developed in its devcontainer.
 ```
 rails s
 ```
-3) POST /certificates to acquire cert in another terminal (need to provide `common_name` param):
+3) POST /certificates to acquire cert (need to provide `common_name` param):
 ```
 curl -X POST http://localhost:3000/certificates \
 -H "Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJqb2huLmRvZUBleGFtcGxlLmNvbSIsIm5hbWUiOiJKb2huIERvZSIsImlhdCI6MTUxNjIzOTAyMiwiZ3JvdXBzIjpbImdyb3VwMSIsImdyb3VwMiJdLCJhdWQiOiJhc3RyYWwifQ.tfRLXmE_eq-piP88_clwPWrYfMAQbCJAeZQI6OFxZSI" \
 -H "Content-type: application/json" \
 -d "{ \"cert_issue_request\": { \"common_name\": \"example.com\" } }"
 ```
-4) Run the tests from devcontainer terminal:
+4) POST and GET /secrets to save and fetch a secret:
+```
+curl -X POST http://localhost:3000/secrets \
```

Comment:
Not a blocker, but can we look into something that will generate docs for us from the code? Rails must provide something like that.

Reply:
There's a story in the backlog to make Openapi / swagger doc -- but I often dislike having a lot of documentation mixed into the source code (as javadoc, rdoc, etc).

Reply:
Can't it extract something from the controllers and allowed_params?